test/persist/test_failure_indices.sh

```diff
@@ -15,31 +15,19 @@ DUMMY_CMD="abci-cli dummy --persist $TMHOME/dummy" # &> dummy_${name}.log"
 function start_procs(){
     name=$1
     indexToFail=$2
-    echo "Starting persistent dummy and tendermint"
-    if [[ "$CIRCLECI" == true ]]; then
-        $DUMMY_CMD &
-    else
-        $DUMMY_CMD &> "dummy_${name}.log" &
-    fi
+    echo "Starting persistent dummy and tendermint: $name"
+    $DUMMY_CMD &> "dummy_${name}.log" &
```

Comment:
will I be able to see this log on jenkins ?

Reply:
working on it

Reply:
yes, they'll be archived as aritfacts, e.g., here's the `coverage.txt` that can be copied by a job that sends it to codecov.